test/parallel/test-file-write-stream3.js

```diff
@@ -199,8 +199,8 @@ const run_test_5 = common.mustCall(function() {
   };
   const err = {
     code: 'ERR_OUT_OF_RANGE',
-    message: 'The value of "start" is out of range. ' +
-             'It must be >= 0 and <= 2 ** 53 - 1. Received 9007199254740992',
+    message: 'The value of "start" is out of range. It must be ' +
+             '>= 0 and <= 2 ** 53 - 1. Received 9,007,199,254,740,992',
```

Comment:
I guess may also argue that a number like `9,007,199,254,740,992` is not any more useful than `9007199254740992` here - it breaks on thousands while the upper limit is displayed with a power of 2, so unless you know what `2 ** 53 - 1` looks like in decimals, or unless `2 ** 53 - 1` is displayed in decimals, you can't really tell which one is bigger. A more useful format may be something displayed in hexadecimals?

Reply:
@joyeecheung I would ideally like to show the difference to the upper or lower bound depending on what is nearer. The problem with that is just that the range is passed through as string and we can not use that for a bounds check. I would like to change that on the long term for suitable cases but it has no high priority at the moment (it's more like a gimmick). That would probably be most useful for the user.

The limit is only a power of two in case it's the buffer ranges error and it it's bigger than `2 ** 32` or below `-(2 ** 32)`.

Hexadecimals can mostly not be read by humans at all and I would not like to introduce those even though they reduce the total length of the input.

I guess this is somewhat personal preference but I prefer `9,007,199,254,740,992` over `9007199254740992`. Is this comment blocking? If it's not, I'd like to land it as is.

Reply:
I don't feel strong enough to block this, since this is just personal preference.

Reply:
Fwiw, adding the commas here also makes the value harder to copy/paste.

Reply:
@addaleax yes, that's clear. @joyeecheung suggested to use the underscore because of that (https://github.com/nodejs/node/pull/27228#issuecomment-483111236).

Do you also prefer that over the comma? (I personally would have changed it after a while but I guess I can also change it right now so that copying works in future versions.)

@richardlau any opinion on that?

Reply:
I initially found the underscores a bit odd to look at, but if they’re likely to become part of the language then I can get behind that. Using commas could possibly lead to someone raising an issue that the number is formatted incorrectly for their locale.

Reply:
I updated it to use underscores from now on.